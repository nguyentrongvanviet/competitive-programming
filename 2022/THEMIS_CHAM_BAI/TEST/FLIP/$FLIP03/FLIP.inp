5 8
2 9
8 9
4 5
4 4
8 1
5
7
8
4
3
4
1
7
