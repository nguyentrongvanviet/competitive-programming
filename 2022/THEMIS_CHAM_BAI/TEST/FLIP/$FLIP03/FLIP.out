9 9 4 4 8 